Formula/casperjs.rb

```diff
@@ -23,36 +14,16 @@ class Casperjs < Formula
     sha256 "fc20c22ec10d175d84bec4dd05d83fa8ea7fc78723a4e8d0f4041d5c00878651" => :mountain_lion
   end
 
-  devel do
-    url "https://github.com/n1k0/casperjs/archive/1.1-beta4.tar.gz"
-    sha256 "144e9a32d8ca677419924f6183aad26e0061d2c08d79abdb998796c010553654"
-    version "1.1-beta4"
-
-    resource "phantomjs" do
-      url "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-macosx.zip"
-      sha256 "8f15043ae3031815dc5f884ea6ffa053d365491b1bc0dc3a0862d5ff1ac20a48"
-    end
-  end
-
-  head do
-    url "https://github.com/n1k0/casperjs.git"
-
-    resource "phantomjs" do
-      url "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-macosx.zip"
-      sha256 "8f15043ae3031815dc5f884ea6ffa053d365491b1bc0dc3a0862d5ff1ac20a48"
-    end
-  end
-
   # For embedded Phantomjs
   depends_on :macos => :snow_leopard
+  depends_on "phantomjs"
 
   def install
     libexec.install Dir["*"]
-    (libexec/"phantomjs").install resource("phantomjs")
 
     (bin/"casperjs").write <<-EOS.undent
       #!/bin/bash
-      export PATH=#{libexec}/phantomjs/bin:$PATH
+      export PATH=#{Formula["phantomjs"].opt_bin}:$PATH
```

Comment:
If it's not using a custom `phantomjs` now & will actually work against our `phantomjs` okay you can just remove the script & go back to a more simple symlink mechanism.